### PR TITLE
packaging stuff

### PR DESCRIPTION
**Pull request by jackeekaplan**
add setup.py, change directory structure
use scripts keyword arg in setup.py with bin/detect-wizard script so that detect-wizard will be installed as a command-line tool

**to test:**
pip3 install -i https://test.pypi.org/simple/ detect-wizard==1.0b10
detect-wizard -i

### CODE REVIEW

**Comment by jackeekaplan** [comment]
Almost all the changes in detect_wizard.py are from my IDE reformatting the file to PEP8 standards

File: setup.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    long_description = fh.read()
+
+setuptools.setup(
+    name="detect_wizard",  # Replace with your own username
+    version="1.0-Beta-10",
+    author="Matthew Brady, Jay Ricco, Jaclyn Kaplan, Damon Weinstein",
+    author_email="w3matt@gmail.com",
+    description="A wizard to run Synopsys Detect",
+    long_description=long_description,
+    long_description_content_type="text/markdown",
```
Comment (damonw-bds) [inline setup.py:13]: It would probably be good for Matthew to write some type of longer description here.

**Review by damonw-bds — APPROVED** [review]
There were a lot of formatting changes, which made it somewhat difficult to see the real changes.  But as far as I can see things look good.

**Review by jayricco — APPROVED** [review]
looks good to me! I second Damon's point about a better description but I really don't think that's a huge issue.

**Pull request merged**